README.md

```diff
@@ -62,7 +62,9 @@ Methods you can call on instances of TreeView, for example on the return value o
 * `reset()`: Resets the treeview to default zoom with centered root.
 * `setFullScreen(isFullScreen)`: Sets the visualisation in full screen mode. Is called with a Boolean as parameter indicating if full screen is to be enabled.
 * `getRoot()`: Returns the root of the visualisation tree. This is a [TreeviewNode](/src/treeview/treeviewNode.js).
-* `update()`: Applies changes made to the tree or its data.
+* `update(TreeviewNode)`: Applies changes made to the tree or its data starting form the given [TreeviewNode](/src/treeview/treeviewNode.js) `Node`.
+* `center(TreeviewNode)`: Centers the given node 

```

Comment:
This mentions `center`, but the code change shows `centerNode`. I support the latter should be used here as well? 

_[Original comment](https://github.ugent.be/unipept/unipept-visualizations/pull/22#discussion_r15237) by @bmesuere on Mon May 07 2018 at 15:37._